packages/sdk/src/engine-commands/formatSchema.ts

```diff
@@ -19,34 +16,32 @@ export async function formatSchema({ schemaPath, schema }: { schemaPath?: string
     throw new Error(`Parameter schema or schemaPath must be passed.`)
   }
 
-  const prismaFmtPath = await resolveBinary(BinaryType.prismaFmt)
-  const showColors = !process.env.NO_COLOR && process.stdout.isTTY
```

Comment:
Is this still supported?

Reply:
This was dead code — reformatting never produces output with terminal colours.